package/network/utils/comgt/files/ncm.sh

```diff
@@ -84,6 +84,7 @@ proto_ncm_setup() {
 	[ -n "$delay" ] && sleep "$delay"
 
 	manufacturer=`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | awk '/Manufacturer/ { print tolower($2) }'`
+	manufacturer=`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | head -2 | tail -1 | tr -d "\r" | tr [A-Z] [a-z]`
```

Comment:
Tell us what the output of "ATI" is. We probably need to modify the manufacturer detection for other models as well.